spec/person_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../person'
+
+describe Person do
+  context 'When testing the Person' do
+    it 'create a new book when person class is initialised' do
+      person = Person.new(34, 'Duane David')
+      id = person.id
+      expect(person.age).to eq 34
+      expect(person.name).to eq 'Duane David'
+      expect(person.id).to eq id
+    end
+  end
+end
```

Comment:
- [ Required ] Please consider adding unit tests for the following methods in the `Person` class. It is part of the project requirements that all methods in a class are tested.
  
  - `can_use_services?`
  - `correct_name`
  - `add_rental(book, date)`

## Project requirements
![Screenshot from 2023-01-28 10-01-38](https://user-images.githubusercontent.com/16392046/215252140-6c868bf0-a5f6-425a-92e1-93fb4a6a2eae.png)